src/Raven.Server/Config/Categories/DatabaseConfiguration.cs

```diff
@@ -149,5 +149,12 @@ public DatabaseConfiguration(bool forceUsing32BitsPager)
         [TimeUnit(TimeUnit.Minutes)]
         [ConfigurationEntry("Databases.RegularCleanupThresholdInMin", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public TimeSetting RegularCleanupThreshold { get; set; }
+        
+        /// <summary>
+        /// specifies the max size of cluster transaction batch to be executed on the database at once
+        /// </summary>
+        [DefaultValue(256)]
+        [ConfigurationEntry("Databases.ClusterTransactionBatchSize", ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
shouldn't this be in `Cluster` ? @grisha-kotler @karmeli87 
shouldn't this have 'Max' in the name?

Reply:
Let's move it to `ClusterConfiguration` and `ClusterTransactionBatchSize`
@haludi